pkg/kubelet/util/swap/swap_util.go

```diff
@@ -0,0 +1,109 @@
+package swap
+
+import (
+	"bytes"
+	"fmt"
+	"k8s.io/klog/v2"
+	"k8s.io/mount-utils"
+	"os"
+	sysruntime "runtime"
+	"strings"
+	"sync"
+)
+
+var (
+	tmpfsNoswapOptionSupported        bool
+	tmpfsNoswapOptionAvailabilityOnce sync.Once
+	swapOn                            bool
+	swapOnErr                         error
+	swapOnOnce                        sync.Once
+	rootDir                           string
+)
+
+const TmpfsNoswapOption = "noswap"
+
+// InitializeRootDir gets a path for a base directory, to which we have write permission.
+// This path will be later use for writing temporary files / mount points / etc.
+func InitializeRootDir(path string) {
+	rootDir = path
+}
+
+func IsTmpfsNoswapOptionSupported(mounter mount.Interface) bool {
+	isTmpfsNoswapOptionSupportedHelper := func() bool {
+		if sysruntime.GOOS == "windows" {
+			return false
+		}
+
+		baseMountDir := rootDir
+		if baseMountDir == "" {
+			klog.ErrorS(fmt.Errorf("swap utils root dir is not initialized, falling back to /tmp"), "")
```

Comment:
```suggestion
			klog.ErrorS(errors.New("swap utils root dir is not initialized, falling back to /tmp"), "")
```

Reply:
Out of curiosity, what's the difference between the two? Why is your suggestion better?

Reply:
A rule of thumb: don't have any format strings to process, then don't use a function that would look for them. Some linters also pick on this. Also, `errors.New()` will be somewhat slightly faster, since no need to check if there is anything to parse, etc.